### PR TITLE
Dyn reconfig start angle max points scan

### PR DESCRIPTION
**Pull request by wsaihopfsg**
For https://github.com/PepperlFuchs/ROS_driver/issues/35#issuecomment-754384255

In the current code, the dynamic reconfiguration for start_angle & max_num_points_scan has no effect because these changes are not communicated to R2300 by `set_scanoutput_config`.

This commit has been tested for R2300 only, and no changes have been made for R2000.

### CODE REVIEW

**Comment by ipa-hsd** [comment]
@wsaihopfsg thanks for the contribution! Could you please resolve the conflicts?

**Comment by wsaihopfsg** [comment]
> @wsaihopfsg thanks for the contribution! Could you please resolve the conflicts?

I am done with this conflict resolution too. Thanks

File: pf_driver/src/pf/pf_interface.cpp [hunk 1]
```diff
@@ -272,52 +272,47 @@ void PFInterface::reconfig_callback_r2000(pf_driver::PFDriverR2000Config& config
...
+        config_.skip_scans = config.skip_scans;
+    } else
+    {
+        protocol_interface_->handle_reconfig(config, level);
+    }
+    pipeline_->set_scanoutput_config(config_);
+    protocol_interface_->set_scanoutput_config(info_.handle,config_);
+    config_ = protocol_interface_->get_scanoutput_config(info_.handle);
+    params_ = protocol_interface_->get_scan_parameters(config_.start_angle);
+    pipeline_->set_scan_params(params_);
```
Comment (ipa-hsd) [inline pf_driver/src/pf/pf_interface.cpp:317]: are these changes only because of indentation? Please take care not to introduce white-spaces.

**Comment by ipa-hsd** [comment]
@wsaihopfsg this PR is behind https://github.com/PepperlFuchs/ROS_driver/pull/51. I have pushed couple of commits to this PR.

**Pull request merged**